src/main/java/seedu/address/logic/parser/EditPropertyCommandParser.java

```diff
@@ -0,0 +1,97 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLIENT_ASKING_PRICE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLIENT_CONTACT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLIENT_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLIENT_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DEADLINE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_POSTAL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REMARK;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TYPE;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.EditPropertyCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new AddPropertyCommand object.
+ */
+public class EditPropertyCommandParser implements Parser<EditPropertyCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddPropertyCommand
+     * and returns an AddPropertyCommand object for execution.
+     * @throws ParseException If the user input does not conform the expected format.
+     */
+    public EditPropertyCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_TYPE, PREFIX_ADDRESS, PREFIX_POSTAL,
+                        PREFIX_DEADLINE, PREFIX_REMARK, PREFIX_CLIENT_NAME, PREFIX_CLIENT_CONTACT,
+                        PREFIX_CLIENT_EMAIL, PREFIX_CLIENT_ASKING_PRICE);
+
+        Index index;
+
+        try {
+            index = ParserUtil.parseIndex(argMultimap.getPreamble());
+        } catch (ParseException pe) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditPropertyCommand.MESSAGE_USAGE), pe);
+        }
+
+        EditPropertyCommand.EditPropertyDescriptor editPropertyDescriptor =
+                new EditPropertyCommand.EditPropertyDescriptor();
```

Comment:
Maybe can do a `import seedu.address.logic.commands.EditPropertyCommand.EditPropertyDescriptor;` on top so that you can directly use `EditPropertyDescriptor` instead of the whole verbose type